pkg/osvscanner/osvscanner.go

```diff
@@ -515,11 +520,17 @@ func scanSBOMFile(r reporter.Reporter, path string, fromFSScan bool) ([]scannedP
 				}
 			}
 
-			slices.SortFunc(packages, func(i, j scannedPackage) int {
+			packages2 := make([]scannedPackage, 0, len(packages))
```

Comment:
rename `packages` and `packages2` to something indicating one is a map and one is a slice.